cpg_workflows/scripts/talos_prep/convert_annotated_vcf_to_ht.py

```diff
@@ -0,0 +1,324 @@
+        logging.info('No MANE table found, skipping annotation - dummy values will be entered instead')
+        return ht.annotate(
+            transcript_consequences=hl.map(
+                lambda x: x.annotate(
+                    mane_status=MISSING_STRING,
+                    ensp=MISSING_STRING,
+                    mane_id=MISSING_STRING,
+                ),
+                ht.transcript_consequences,
+            ),
+        )
+
+    # read in the mane table
+    with open(mane_path) as handle:
+        mane_dict = json.load(handle)
+
+    # convert the dict into a Hail Dict
+    hl_mane_dict = hl.dict(mane_dict)
+
+    key_set = hl_mane_dict.key_set()
+
+    # annotate the variant with MANE data, recovering Mane status, NM ID, and ENSP ID
+    return ht.annotate(
+        transcript_consequences=hl.map(
+            lambda x: x.annotate(
+                mane_status=hl.if_else(
+                    key_set.contains(x.transcript),
+                    hl_mane_dict[x.transcript]['mane_status'],
+                    MISSING_STRING,
+                ),
+                ensp=hl.if_else(
+                    key_set.contains(x.transcript),
+                    hl_mane_dict[x.transcript]['ensp'],
+                    MISSING_STRING,
+                ),
+                mane_id=hl.if_else(
+                    key_set.contains(x.transcript),
+                    hl_mane_dict[x.transcript]['mane_id'],
+                    MISSING_STRING,
+                ),
+            ),
+            ht.transcript_consequences,
+        ),
+    )
+
+
+def cli_main():
+    """
+    take an input VCF and an output MT path
+    also supply the alpha_missense table created by parse_amissense_into_ht.py
+    """
+
+    parser = ArgumentParser(description='Takes a BCSQ annotated VCF and makes it a HT')
+    parser.add_argument('--input', help='Path to the annotated sites-only VCF', required=True)
+    parser.add_argument('--output', help='output Table path, must have a ".ht" extension', required=True)
+    parser.add_argument('--gene_bed', help='BED file containing gene mapping')
+    parser.add_argument('--am', help='Hail Table containing AlphaMissense annotations', required=True)
+    parser.add_argument('--mane', help='Hail Table containing MANE annotations', default=None)
+    parser.add_argument('--checkpoint_dir', help='directory to checkpoint to', default=None)
+    args = parser.parse_args()
+
+    assert args.output.endswith('.ht'), 'Output path must end in .ht'
+
+    main(
+        vcf_path=args.input,
+        output_path=args.output,
+        gene_bed=args.gene_bed,
+        alpha_m=args.am,
+        mane=args.mane,
+        checkpoint_dir=args.checkpoint_dir,
+    )
+
+
+def main(
+    vcf_path: str,
+    output_path: str,
+    gene_bed: str,
+    alpha_m: str,
+    mane: str | None = None,
+    checkpoint_dir: str | None = None,
+):
+    """
+    Takes a VEP-annotated VCF, reorganises into a Talos-compatible MatrixTable
```

Comment:
VEP?

Reply:
Aye there's a bit of copy-paste carryover here, this was previously the VEP VCF to MT script

Reply:
Aye there's a bit of copy-paste carryover here, this was previously the VEP VCF to MT script